engine/crates/federated-graph/src/render_sdl/render_federated_sdl.rs

```diff
@@ -28,62 +28,57 @@ pub fn render_federated_sdl(graph: &FederatedGraphV3) -> Result<String, fmt::Err
         sdl.push('\n');
     }
 
-    let query_type_exists = graph.objects.iter().any(|object| &graph[object.name] == "Query");
-    if !query_type_exists {
-        writeln!(sdl, "type Query\n")?;
-    }
-
     for (idx, object) in graph.objects.iter().enumerate() {
         let object_name = &graph[object.name];
         let is_query_root = graph.root_operation_types.query == ObjectId(idx);
 
-        if let Some(description) = object.description {
-            write!(sdl, "{}", Description(&graph[description], ""))?;
-        }
+        let mut fields = graph[object.fields.clone()]
+            .iter()
+            .filter(|field| !(is_query_root && ["__type", "__schema"].contains(&graph[field.name].as_str())))
```

Comment:
The engine just filters out any fields starting w/ a `__`.  Not a big deal but do you think it's worth doing that here as well?

Reply:
This was meant to have a link: https://github.com/grafbase/grafbase/blob/main/engine/crates/engine/src/registry/export_sdl.rs#L66

Reply:
Since the rendered SDL is visible in the dashboard, I think it's better to leave these out (same as __type and __schema), but it's not super important.

Reply:
Yeah, I was just suggesting an alternative method for leaving them out.  But doesn't matter.

Reply:
Ah, you mean feed the schema to engine-v2, then use the introspection machinery to render the api schema as sdl?

Reply:
Apparently I chose the worlds least clear way to suggest this:

```suggestion
            .filter(|field| !field.starts_with("__"))
```

Reply:
aah ok, yes, let's do that :+1: